docs/README.md

```diff
@@ -98,7 +98,7 @@ You can then access the html files at `http://localhost/index.html`.
 Updating content in the [Commands Reference](https://hyperledger-fabric.readthedocs.io/en/latest/command_ref.html) topic requires additional steps. Because the information in the Commands Reference topic is generated content, you cannot simply update the associated markdown files.
 - Instead you need to update the `_preamble.md` or `_postscript.md` files under `src/github.com/hyperledger/fabric/docs/wrappers` for the command.
 - To update the command help text, you need to edit the associated `.go` file for the command that is located under `/fabric/internal/peer`.
-- Then you need to run the command `make help-docs` which generates the updated markdown files under `docs/source/commands`. **Tip:** Before running `make help-docs`, ensure that you have the [Go Programming language](/source/prereqs.html#go-programming-language) installed.
+- Then you need to run the command `make help-docs` which generates the updated markdown files under `docs/source/commands`. **Tip:** Before running `make help-docs`, ensure that you have the [Go Programming language](/source/prereqs.html#go-programming-language) installed and that your [GOPATH](/source/dev-setup/devenv.html#go-programming-language) is configured.
```

Comment:
Ran this command successfully. 

1. You do not need to set the GOPATH to run the commands, you can run without.
2. It may be worth pointing out that this command is run from the `fabric` folder. This section is below commands that are run from the `docs` folder.

Reply:
Removed the GOPATH instructions and added that you have to run the command from the fabric folder.